### ISSUE TITLE
ENOENT, no such file or directory 'undefined/

### ISSUE THREAD

**Issue by t4t5** [abuiles/ember-watson#47]
When running `ember watson:convert-ember-data-model-lookups`, I get this:

```
ENOENT, no such file or directory 'undefined/'
Error: ENOENT, no such file or directory 'undefined/'
    at Error (native)
    at Object.fs.readdirSync (fs.js:761:18)
    at walkSync (/Users/Tristan/dev/Ludu/node_modules/ember-watson/node_modules/walk-sync/index.js:14:20)
    at module.exports (/Users/Tristan/dev/Ludu/node_modules/ember-watson/lib/helpers/find-files.js:10:13)
    at EmberWatson.transformEmberDataModelLookups (/Users/Tristan/dev/Ludu/node_modules/ember-watson/lib/watson.js:27:15)
    at Class.module.exports.run (/Users/Tristan/dev/Ludu/node_modules/ember-watson/addon/commands/convert-ember-data-model-lookups.js:16:12)
    at Class.<anonymous> (/Users/Tristan/dev/Ludu/node_modules/ember-cli/lib/models/command.js:136:17)
    at lib$rsvp$$internal$$tryCatch (/Users/Tristan/dev/Ludu/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:489:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/Tristan/dev/Ludu/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:501:17)
    at lib$rsvp$$internal$$publish (/Users/Tristan/dev/Ludu/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:472:11)
```

Could it have anything to do with the fact that I'm using a POD structure?

**Comment by abuiles** [comment]
@t4t5 which version are you running?  Can you try with 0.5.3

**Comment by t4t5** [comment]
I'm using 0.5.3, together with Ember 2.0.0-beta.1 and Ember Data 1.13.0.

**Comment by jcope2013** [comment]
same

**Comment by abuiles** [comment]
Just published ember-watson@0.5.5 which includes a fix to this issue, thanks to @marcioj for fixing it.
